client/generate_proxy.go

```diff
@@ -0,0 +1,92 @@
+// +build ignore
```

Comment:
You'll need license headers on these files (for the generated code you might need to alter `/internal/cmd/tools/copyright/licensegen.go` if you don't want to add a header there)